packages/components/src/panel/body.js

```diff
@@ -61,8 +61,8 @@ class PanelBody extends Component {
 									<G><Path d="M7.41,8.59L12,13.17l4.59-4.58L18,10l-6,6l-6-6L7.41,8.59z" /></G>
 								</SVG>
 							}
-							{ icon && <Dashicon icon={ icon } className="components-panel__icon" /> }
 							{ title }
+							{ icon && <RawIcon icon={ icon } className="components-panel__icon" size={ 20 } /> }
```

Comment:
What's the impact of this change on other usage of `PanelBody` with icons? Are we ok with all panel body icons being moved to the right or should it be specific to the inserter?

Reply:
Last time I checked, the only usages were in the inserter, and this PR has verified them all. 

Whether we're planning to use the `PanelBody` for something else, and icons would make sense on the right in these cases - perhaps @mtias @jasmussen would know better.

Reply:
It looks like it was introduced explicitly for this particular usage (Inserter), so yes, I think I'm fine with the change.

Reply:
Yes, it was added for the reusable block feature.